serializers/avro/src/test/java/io/pravega/schemaregistry/serializer/avro/impl/SerializerTest.java

```diff
@@ -52,8 +52,8 @@ public void testAvroSerializers() {
         SerializerConfig config = SerializerConfig.builder().registryClient(client).groupId("groupId").build();
         AvroSchema<Test1> schema1 = AvroSchema.of(Test1.class);
         AvroSchema<Test2> schema2 = AvroSchema.of(Test2.class);
-        VersionInfo versionInfo1 = new VersionInfo("name", 0, 0);
-        VersionInfo versionInfo2 = new VersionInfo("name", 1, 1);
+        VersionInfo versionInfo1 = new VersionInfo("name", SerializationFormat.Avro.getFullTypeName(), 0, 0);
```

Comment:
firstly, none of our apis dont take version info as input.. so that is not a problem per se.
secondly, our apis are marked experimental so we are not promising compatibility. however, as i said, the api compatibility is not broken because of versionInfo change. 
However, its broken for two api changes where we allowed users to get or delete schema version using path params type and version number. now they need to provide type format and version number in path params.